backend/engine/plugins/gitlab_repo_health/README.md

```diff
@@ -0,0 +1,310 @@
+        },
+        {
+            "type": "branch_status_checks",
+            "id": "branch_ci_required",
+
+            # 'name' and 'description' are optional and will override the defaults.
+            # They can be useful for adding additional context to rules
+            #
+            # In some cases, it might make sense to run multiple checks with the same type. In those cases,
+            # overriding the id, name, and description is strongly encouraged
+            "name": "Branch - Continuous Integration Required",
+            "description": "Require that the CI jobs pass before pull requests can be merged",
+
+            # 'required_checks' is a rule-specific configuration. Many rules have these.
+            # You can examine the config schema for rules with `repo-health --list-available-rules`
+            "checks": {
+                "all_of": [
+                    "ci/build",
+                    "ci/test"
+                ]
+            }
+        },
+        {
+            "type": "branch_pull_requests",
+
+            # 'expect' is a common rule-specific configuration. It allows the config to specify more in-depth
+            # configuration requirements by comparing the included keys and values directly against
+            # the API response from Github.
+            #
+            # To figure out what values can be expected, go to `src/rules/<rule id>.py`.
+            # The comments in that file will specify what Github API response it compares to
+            "expect": {
+                "dismiss_stale_reviews": true,
+                "require_code_owner_reviews": true,
+            }
+        }
+    ]
+}
+```
+
+### Omitted Fields
+
+Any field can be omitted. If a field is omitted, it will not factor into the
+computation for the rule. By contrast, if a field is set to `false` in a config,
+it may be explicitly required to be `false`, even if it's unintuitive for a
+configuration to require the field to be false.
+
+For example, the follow config would require that `dismiss_stale_reviews` is false.
+
+```json
+{
+    "type": "branch_pull_requests",
+    "expect": {
+        "dismiss_stale_reviews": false,
+        ...
+    }
+}
+```
+
+It does not make much sense to do this from a security configuration
+perspective. It would, however, make sense to allow any value there so that
+teams can decide what is best for them. If the intent is to have the pull
+requests rule ignore `dismiss_stale_reviews`, it should be omitted:
+
+```json
+{
+    "type": "branch_protection_pull_requests",
+    "expect": {
+        ...
+    }
+}
+```
+
+## Rules
+
+Rules are checks that are run on the repositories.
+
+### Errors
+
+If an error occurs while running a rule, it will automatically fail. In addition, its result will have an `error_message` field that contains information on what went wrong.
+
+### Composite Rules
+
+Composite rules are new in version `1.0.0`. They are rules that are made up of other rules, rather
```

Comment:
Does the versioning stuff make sense? The Github healthcheck plugin is installable as a Python project. Since the GitLab version got rid of that, it may not make sense naymore